modules/generic-extras/src/main/scala/io/circe/generic/extras/decoding/ReprDecoder.scala

```diff
@@ -34,10 +34,10 @@ abstract class ReprDecoder[A] extends Decoder[A] {
     name: String,
     defaults: Map[String, Any]
   ): Decoder.Result[B] = result match {
-    case r @ Right(_) => r
-    case l @ Left(_) => defaults.get(name) match {
+    case r @ Right(_) if r ne Decoder.keyMissingNone => r
```

Comment:
Why not just `case r @ Right(v) if v.isEmpty => r`?

Reply:
@travisbrown 
Can I have your moment to get review?
Perhaps, you may missed my response due to last GitHub outage. 
Thanks in advance 😃